docs/reference/commandline/cli.md

```diff
@@ -78,6 +78,11 @@ For example:
 Instructs Docker to use the configuration files in your `~/testconfigs/`
 directory when running the `ps` command.
 
+> **Note:**
+> Calling docker with `sudo` results in Docker looking first in `/root/.docker/`.
+> If it isn't found it looks for the user who did the `sudo` call and resolves the
```

Comment:
mmm `for the current user` is wrong because it specifically resolves for the current *sudo* user